### PR TITLE
[#116] Add clarification to hosts_config.json (4-2-stable)

### PR DESCRIPTION
**Pull request by alanking**
I don't know if this is necessarily true. I would expect a FQDN to work as well.

### CODE REVIEW

File: docs/system_overview/configuration.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ The `local` entry can define multiple addresses for the local server.
 
 The `remote` entries each define a remote server with a set of aliases.
 
-The first address in each stanza is the preferred address and will be used for connecting to clients and remote servers.  The additional addresses are the aliases.
+The first address in each stanza is the preferred address and will be used for connecting to clients and remote servers. This first address should be the hostname of the machine in question and is expected to match the result of running `hostname` on said host. The additional addresses are the aliases.
```
Comment (trel) [inline docs/system_overview/configuration.md:153]: two suggestions...

`s/machine/server/`

`s/said host/that server/`

and I think we're good.

Comment (alanking) [reply docs/system_overview/configuration.md]: Done

**Comment by alanking** [comment]
#'d

**Comment by trel** [comment]
and please to cherry-pick to master

**Pull request merged**